### ISSUE TITLE
Attachments not working.

### ISSUE THREAD

**Issue by Panoplos** [os/slacker#30]
I am trying to post the following via `slacker`:

```python
    attachment = {
        "attachments": [
            {
                "fallback": "Fallback text",
                "title": "Title",
                "fields": [
                    {
                        "value": "*some field data*",
                        "short": True
                    },
                    {
                        "value": "*some more data*",
                        "short": True
                    }
                ],
                "mrkdwn_in": ["fields"],
                "color": "good"
            }
        ]
    }
    slacker.chat.post_message(
        SLACK_CHANNEL,
        "Testing Attachments",
        SLACK_USER,
        attachments=attachment,
        icon_url=SLACK_ICON_URL
    )
```

Where I have set the constants to my settings, but you should be able to test this yourself.

![image](https://cloud.githubusercontent.com/assets/2549985/7079836/598cf806-df64-11e4-856f-480538f8710e.png)

Note, if I use `json.dumps(attachment)`, the result is:

![image](https://cloud.githubusercontent.com/assets/2549985/7079854/98d48812-df64-11e4-830b-b30542165433.png)

Which contains empty attachments.

**Comment by gavinhodge** [comment]
I had the same problem with a similar library, and the solution was to `json.dumps(attachments)`.

Just be sure your attachments variable is a *list* not a dict. The library includes the `"attachments":` part for you. I hope this helps.

**Comment by Jonovono** [comment]
Ya json.dumps worked for me. I might submit a PR tomorrow. Feel like this should be handled.

**Comment by os** [comment]
Fixed in #35. Closing.

**Comment by os** [comment]
Thanks for the ticket @Panoplos.
